readme.md

```diff
@@ -126,24 +138,23 @@ You can read the annotations for all the details of how Marionette works, and ad
 MarionetteJS currently works with the following libraries:
 
 * [jQuery](http://jquery.com) v1.8+
-* [Underscore](http://underscorejs.org) v1.4.4 - 1.6.0
-* [Backbone](http://backbonejs.org) v1.0.0 - 1.1.2 are preferred.
-* [Backbone.Radio](https://github.com/marionettejs/backbone.radio) Comes automatically with the bundled build.
-* [Backbone.BabySitter](https://github.com/marionettejs/backbone.babysitter) Comes automatically with the bundled build.
+* [Underscore](http://underscorejs.org) v1.8.3
+* [Backbone](http://backbonejs.org) v1.3.3
+* [Backbone.Radio](https://github.com/marionettejs/backbone.radio) Comes bundled (v1.0.0)
+* [Backbone.BabySitter](https://github.com/marionettejs/backbone.babysitter) Comes bundled (v1.0.0)
```

Comment:
I didn't actually remove Radio as a dependency in package.json.  I still kinda think it's cool to keep it as default.  People can alias it out if need be right?  But `<script>` tag users can add Mn without having to worry about Radio day one.